src/core/core.scale.js

```diff
@@ -1255,16 +1264,20 @@ class Scale extends Element {
 			ctx.fillStyle = tickFont.color;
 			ctx.textBaseline = 'middle';
 			ctx.textAlign = item.textAlign;
+			ctx.strokeStyle = optionTicks.strokeStyle;
```

Comment:
I think this should be `tickFont.strokeStyle`, I guess this is the missing link.

Reply:
Hi @kurkle, thanks for the suggestion, but unfortunately nop:

![image](https://user-images.githubusercontent.com/676681/69749992-f93d8780-114b-11ea-8ba4-15cd20eaccef.png)

I need to explicitly add it to `_parseFont` to have it available in `tickFont`, but it seems that I'm adding this new options to everywhere that needs to parse fonts, so it's like a little dirty, doesn't it?

However, I can't find how to use the options directly inside `_drawLabels`, the new arguments are empty if I don't add in `_parseFont` or `parseTickFontOptions` .. so there is some magic somewhere that I can't find

Reply:
You had those changes in `parseFontOptions` when debugging?

Reply:
yep, it's there `lineWidth`, `strokeStyle`:
```
function parseFontOptions(options, nestedOpts) {
	return helpers.extend(helpers.options._parseFont({
		fontFamily: valueOrDefault(nestedOpts.fontFamily, options.fontFamily),
		fontSize: valueOrDefault(nestedOpts.fontSize, options.fontSize),
		fontStyle: valueOrDefault(nestedOpts.fontStyle, options.fontStyle),
		lineHeight: valueOrDefault(nestedOpts.lineHeight, options.lineHeight),
		lineWidth: valueOrDefault(nestedOpts.lineWidth, options.lineWidth),
		strokeStyle: valueOrDefault(nestedOpts.strokeStyle, options.strokeStyle),
	}), {
		color: resolve([nestedOpts.fontColor, options.fontColor, defaults.global.defaultFontColor])
	});
}
```

Reply:
Finally, the options are in `optionTicks` and as I commented above, I let also the `pareFontOptions` for the minor and major.

Reply:
I'd be ok with enabling it in `_parseFont` if we allow all the text to be stroked as well (likely outside the scope of this PR)

Reply:
I don't think specifying different lineWidth for major ticks works now.